docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/advanced-usage/creating-a-whitelabel-build/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Creating a whitelabel build"
+date: "2022-09-14"
+sidebar_position: 4000
+---
+
+```mdx-code-block
+import Block5966 from "@site/docs/reusable/javascript-tracker-release-badge-v3/_index.md"
+
+<Block5966/>
+```
+
+Every time the snowplow tracker loads on the page, a queue-like object `GlobalSnowplowNamespace` is initialised on the global window scope. This object is then used to queue commands before the tracker snippet (sp.js) has fully loaded.
+
+In some cases, like two Snowplow instances on the same page from different sources or different tracker versions, there might be a need to separate this queue to avoid conflicts and tracking inconsistencies. To help you with this, we provide a custom build option for our JavaScript tracker that will produce modified loader tags (tag.js & tag.min.js) and tracker snippets (sp.js & sp.lite.js).
+
+To create the _whitelabel_ build, as we call it, you need to follow the steps as shown below:
+
+1. Clone the JavaScript repository `git clone git@github.com:snowplow/snowplow-javascript-tracker.git`
+2. Go through the [Maintainer quick start](https://github.com/snowplow/snowplow-javascript-tracker#maintainer-quick-start) to setup the required tooling.
+3. Change directory to the JavaScript tracker project `cd ./trackers/javascript-tracker`.
+4. Run the custom whitelabel build command `rushx build --whitelabel=MyWhitelabel` replacing MyWhitelabel with the namespace you want.
```

Comment:
It’s not entirely clear what will happen. Will `GlobalSnowplowNamespace` become `GlobalMyWhitelabelNamespace`? I would make that explicit.

Reply:
Added this on 4:

4. Run the custom whitelabel build command `rushx build --whitelabel=MyWhitelabel` replacing MyWhitelabel with the namespace you want. This will replace all references of `GlobalSnowplowNamespace` with `MyWhitelabel`.